.github/workflows/test_and_deploy.yml

```
@@ -58,47 +57,61 @@ jobs:
           retention-days: 7
 
   test_pyver:
-    runs-on: ubuntu-latest
     needs: build
     strategy:
       matrix:
-        pyver:
-          - 3.7
-          - 3.8
+        pyver: [3.6, 3.7, 3.8]
```

Latest numba doesn't support 3.6. And let's add new python versions
```suggestion
        pyver: [3.7, 3.8, 3.9, 3.10]
```